andela/travel_tool_front
# #161758633 Disable change room if user is checked out

#### What does this PR do?
- Disable change room if trip status changes to checked out

#### Description of Task to be completed?
- Include a condition that checks if the checkStatus of a trip is set to `checkOut`
- Do not display the `change room` button if trip is checked out

#### How should this be manually tested?

- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Enter into the project directory - `cd travel_tool_front`
- Checkout to this branch - `bg-disable-change-room-161758633`
- Install dependencies and start the app - make start
- Start the backend -make start
- Navigate to Travela in your browser and login
- Go to theDB and on the UserRoles table, set yourself as a `Travel Admin` as well as `Manager`
- On your browser, refresh the page, Navigate to the Residence tab
- Create a new request on Travela and have it approved
- On the Residence tab, go to guesthouses to be able to view the allocation of rooms
- On the trip you've just created, when you open the trip as travel admin can see the `Change Room` button
- Go back to the Residence tab, click on `Check in` and subsequently `Check out`
- Back on the Residence tab, go to guesthouses to be able to view the allocation of rooms
- On the trip you've just checked out, when you open the trip you should not be able to see `Change Room`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161758633](https://www.pivotaltracker.com/story/show/161758633)

#### Screenshots (if appropriate)
N/A

#### Questions:
